src/Servers/HttpSys/src/AuthenticationManager.cs

```diff
@@ -63,6 +63,19 @@ public bool AllowAnonymous
     /// </summary>
     public string? AuthenticationDisplayName { get; set; }
 
+    /// <summary>
+    /// If true, the Kerberos authentication credentials are cached. Kerberos or Negotiate
```

Comment:
I agree with @blowdart that it would be nice to give some indication of the lifetime of the cached credentials.

Reply:
> I agree with @blowdart that it would be nice to give some indication of the lifetime of the cached credentials.

I made a slight adjustment of the docstring in db5b58074c51ee297c1ff5d2ec67bcf0c6a40515 to address this part. What do you think about the new description?

Reply:
To confirm, that's "subsequent anonymous requests _on the same connection_"?

Reply:
> To confirm, that's "subsequent anonymous requests _on the same connection_"?

Yes. I tweaked the docstring accordingly in e2a327a6df7cec7ac7df37b59ff81e0f9309a89b.